lib/vagrant-service-manager/command.rb

```diff
@@ -221,6 +229,28 @@ def print_vagrant_box_version(script_readable = false)
           end
         end
       end
+
+      def restart_service(service)
+        sccli_services = ["openshift", "kubernetes", "k8s"]
```

Comment:
Lets extract this as `SCCLI_SERVICES` inside `ServiceManager` namespace(Module) like https://github.com/projectatomic/vagrant-service-manager/blob/master/lib/vagrant-service-manager/config.rb#L5